meta-mender-tegra/recipes-bsp/u-boot/libubootenv-fake/fw_printenv

```diff
@@ -9,26 +18,31 @@ if [ "$1" = "-n" ]; then
     fi
 fi
 if [ -z "$1" ]; then
-    echo "mender_boot_part=`nvbootctrl get-current-slot`"
+    bootpart=`get_bootpart`
+    echo "mender_boot_part=$bootpart"
     exit 0
 fi
 while [ -n "$1" ]; do
     case "$1" in
-        mender_boot_part|mender_boot_part_hex)
-            [ -n "$quiet" ] || echo -n "$1="
-            nvbootctrl get-current-slot
+        mender_boot_part)
+            bootpart=`get_bootpart`
+            [ -n "$quiet" ] || echo "$1=$bootpart"
```

Comment:
The `-n` flag on `fw_setenv` is supposed to print the value without the `<variable>=` prefix, which is why all of the checks for `$quiet` were on a separate line from the actual output.

Mender probably doesn't ever use that flag, but changing the logic of that check isn't really correct.

Reply:
I've pushed a new commit reverting these changes. Note that I did fix tab / space indentation in the "upgrade_available" case, so there are still some changes there.

Reply:
Thanks